include/modem/lte_lc.h

```diff
@@ -967,101 +1259,113 @@ int lte_lc_connect(void);
  */
 int lte_lc_init_and_connect(void);
 
-/** @brief Connect to LTE network. Non-blocking.
+/**
+ * Connect to LTE network.
+ *
+ * The function returns immediately.
  *
  * @note The module must be initialized before this function is called.
  *
- * @param handler Event handler for receiving LTE events. The parameter can be
- *	          NULL if an event handler is already registered.
+ * @param handler Event handler for receiving LTE events. The parameter can be @c NULL if an event
+ *                handler is already registered.
  *
  * @retval 0 if successful.
  * @retval -EINVAL if no event handler was registered.
  * @retval -EFAULT if an AT command failed.
  */
 int lte_lc_connect_async(lte_lc_evt_handler_t handler);
 
-/** @brief Initializes the LTE module, configures the modem and connects to LTE
- *	   network. Non-blocking.
+/**
+ * Initializes the LTE module, configures the modem and connects to LTE network.
+ *
+ * The function returns immediately.
  *
- * @note The module can be initialized only once, and repeated calls will
- *	 return 0. lte_lc_connect() should be used on subsequent calls.
+ * @note The module can be initialized only once, and repeated calls will return @c 0.
+ *       lte_lc_connect() should be used on subsequent calls.
  *
- * @param handler Event handler for receiving LTE events. The parameter can be
- *		  NULL if an event handler is already registered.
+ * @param handler Event handler for receiving LTE events. The parameter can be @c NULL if an event
+ *                handler is already registered.
  *
  * @retval 0 if successful.
  * @retval -EFAULT if an AT command failed.
  * @retval -EINVAL if no event handler was registered.
  */
 int lte_lc_init_and_connect_async(lte_lc_evt_handler_t handler);
 
-/** @brief Deinitialize the LTE module, powers of the modem.
+/**
+ * Deinitialize the LTE module, powers of the modem.
```

Comment:
I just now noticed that there's a typo ("of" instead of "off"). But, maybe the whole sentence would be better like this:

"Deinitialize the library and power off the modem."

I will also change other uses of the word "module", because usually LTE module means something different. I think "library" is a better term here.

Reply:
Damn, I missed the typo. Keep cleaning ;).